Kconfig

```diff
@@ -11,4 +11,8 @@ menu "wolfSSL"
         bool "Enable ALPN (Application Layer Protocol Negotiation) in wolfSSL"
         default y
 
+    config WOLFSSL_HAVE_OCSP
+        bool "Enable OCSP (Online Certificate Status Protocol) in wolfSSL"
+        default n
```

Comment:
Can you add a help section here that mentions that this config enables OCSP in Stapling mode